bsp/boards/openmote-b/leds.c

```diff
@@ -59,34 +59,34 @@ uint8_t leds_error_isOn(void) {
     return (uint8_t)(ui32Toggle & BSP_LED_1)>>4;
 }
 
-// orange
```

Comment:
why do youhave to chnage the LEDs?!?

Reply:
just used the version from the subghz project as is